pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -60,23 +60,22 @@ def prettify(html: str) -> str:
 
 # TESTS FOR NOT IMPLEMENTTED FEATURES
```

Comment:
Since `test_rst_directive_abnomitions()` now tests an implemented feature, it should be moved above this marker.

Alternatively, the comment could be moved to the top of the file and changed to say that "xfail" tests are for non-implemented features.